src/util.cpp

```diff
@@ -484,6 +484,24 @@ const boost::filesystem::path &ZC_GetParamsDir()
     return path;
 }
 
+// Return the user specified export directory.  Create directory if it doesn't exist.
+// If user did not set option, or if the path is not a directory, return an empty path.
+const boost::filesystem::path GetExportDir()
+{
+    namespace fs = boost::filesystem;
+    fs::path path;
```

Comment:
Unclear from this whether `path` defaults to empty or not (c/f several lines later, where `path` is explicitly re-set to empty). But per my above comment, this should be moot.